app/controllers/candidate_interface/references/selection_controller.rb

```diff
@@ -62,7 +63,10 @@ def complete
         if @section_complete_form.not_completed?
           @section_complete_form.save(current_application, :references_completed)
           redirect_to candidate_interface_application_form_path
-        elsif current_application.selected_enough_references?
+        elsif !current_application.selected_enough_references?
+          flash[:warning] = "You must select two references to complete this section"
+          render :review
+        else
```

Comment:
Should ever be able to visit this page if you don’t have 2 selected references? I think in this case, we should redirect you to the section start page, which already has content about needing to select 2 references.

Reply:
This is in place for when you land there after deleting a selected reference. 

There is/will be a [private method](https://github.com/DFE-Digital/apply-for-teacher-training/pull/4852/files#diff-691fbbf565a6b43c0f77a4b5d263bba008f73d25169eba5b3b302942b2d33acfR77-R79) that will redirect you away from being able to reach the selected references review page in any other circumstance. It's currently commented out whilst I work on combining the two – provided it's decided that we keep some form of warning in place?

Reply:
![image](https://user-images.githubusercontent.com/47917431/120327715-f8a62400-c2e1-11eb-8862-064fc5df4148.png)